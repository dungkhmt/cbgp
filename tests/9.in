50 50
50 100
32 49
43 39
46 24
24 48
43 34
45 8
34 4
33 44
6 0
28 4
36 7
36 13
17 5
46 11
36 9
42 6
15 8
20 1
4 5
4 6
8 5
18 17
35 11
12 17
47 11
38 20
45 14
49 4
41 46
30 21
6 2
31 1
22 16
20 27
8 9
23 17
31 12
29 24
13 26
48 39
29 3
25 19
22 2
30 19
12 44
45 10
40 16
31 21
37 12
1 2
45 3
14 3
37 0
49 2
42 22
35 14
15 9
33 17
5 34
42 16
13 1
8 1
14 11
42 27
19 0
45 7
6 1
29 11
30 25
23 37
39 32
32 4
21 19
18 23
17 0
31 19
31 0
46 29
2 3
23 12
26 7
43 32
48 11
13 9
42 20
26 3
35 47
9 7
41 48
45 9
33 31
39 2
33 8
33 12
13 7
49 39
38 42
23 0
46 48
32 34
1
48 8 10 5 34 22 38 26 62 16 57 0 42 14 36 28 2 0 33 31 0 0 48 30 15 2 32 27 43 31 31 28 37 18 55 1 51 3 43 12 28 15 45 9 34 19 51 4 48 26 42 10 37 27 33 15 76 1 44 26 44 10 12 4 55 26 9 1 82 0 44 32 34 29 50 5 31 18 53 25 35 20 49 27 32 17 55 27 12 3 37 37 48 29 45 33 52 26 54 24 
