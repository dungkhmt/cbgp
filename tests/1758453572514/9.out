(32, 49), (43, 39), (46, 24), (24, 48), (43, 34), (45, 8), (34, 4), (33, 44), (6, 0), (28, 4), (36, 7), (36, 13), (17, 5), (46, 11), (36, 9), (42, 6), (15, 8), (20, 1), (4, 5), (4, 6), (8, 5), (18, 17), (35, 11), (12, 17), (47, 11), (38, 20), (45, 14), (49, 4), (41, 46), (30, 21), (6, 2), (31, 1), (22, 16), (20, 27), (8, 9), (23, 17), (31, 12), (29, 24), (13, 26), (48, 39), (29, 3), (25, 19), (22, 2), (30, 19), (12, 44), (45, 10), (40, 16), (31, 21), (37, 12), (1, 2), (45, 3), (14, 3), (37, 0), (49, 2), (42, 22), (35, 14), (15, 9), (33, 17), (5, 34), (42, 16), (13, 1), (8, 1), (14, 11), (42, 27), (19, 0), (45, 7), (6, 1), (29, 11), (30, 25), (23, 37), (39, 32), (32, 4), (21, 19), (18, 23), (17, 0), (31, 19), (31, 0), (46, 29), (2, 3), (23, 12), (26, 7), (43, 32), (48, 11), (13, 9), (42, 20), (26, 3), (35, 47), (9, 7), (41, 48), (45, 9), (33, 31), (39, 2), (33, 8), (33, 12), (13, 7), (49, 39), (38, 42), (23, 0), (46, 48), (32, 34), 
numberIntersection=0 
minDistance=2.8284271247461903 
minAngle=0.0517710787684198 
sumAngle=46.108638354362114 
sumDistance=289672.9182157677 
minEdgeLength=3.0 
sumEdgeLength=3081.5114292472213 

0.0000000000 2.8284271247 0.0517710788 46.1086383544 289672.9182157677 3.0000000000 3081.5114292472
0: (97, 166),
1: (15, 177),
2: (66, 97),
3: (74, 90),
4: (109, 119),
5: (113, 197),
6: (93, 137),
7: (70, 84),
8: (0, 198),
9: (59, 78),
10: (141, 0),
11: (97, 74),
12: (46, 182),
13: (63, 91),
14: (84, 76),
15: (58, 94),
16: (68, 124),
17: (110, 194),
18: (99, 183),
19: (86, 149),
20: (43, 139),
21: (91, 163),
22: (65, 124),
23: (97, 175),
24: (93, 92),
25: (84, 161),
26: (70, 88),
27: (51, 139),
28: (144, 154),
29: (75, 93),
30: (87, 161),
31: (20, 185),
32: (114, 106),
33: (10, 195),
34: (174, 122),
35: (86, 72),
36: (65, 85),
37: (94, 171),
38: (48, 134),
39: (106, 98),
40: (68, 121),
41: (97, 90),
42: (66, 130),
43: (115, 96),
44: (21, 188),
45: (66, 44),
46: (94, 82),
47: (86, 68),
48: (105, 95),
49: (104, 107),
97 166 15 177 66 97 74 90 109 119 113 197 93 137 70 84 0 198 59 78 141 0 97 74 46 182 63 91 84 76 58 94 68 124 110 194 99 183 86 149 43 139 91 163 65 124 97 175 93 92 84 161 70 88 51 139 144 154 75 93 87 161 20 185 114 106 10 195 174 122 86 72 65 85 94 171 48 134 106 98 68 121 97 90 66 130 115 96 21 188 66 44 94 82 86 68 105 95 104 107 
0: (97, 166), 1: (15, 177), 2: (66, 97), 3: (74, 90), 4: (109, 119), 5: (113, 197), 6: (93, 137), 7: (70, 84), 8: (0, 198), 9: (59, 78), 10: (141, 0), 11: (97, 74), 12: (46, 182), 13: (63, 91), 14: (84, 76), 15: (58, 94), 16: (68, 124), 17: (110, 194), 18: (99, 183), 19: (86, 149), 20: (43, 139), 21: (91, 163), 22: (65, 124), 23: (97, 175), 24: (93, 92), 25: (84, 161), 26: (70, 88), 27: (51, 139), 28: (144, 154), 29: (75, 93), 30: (87, 161), 31: (20, 185), 32: (114, 106), 33: (10, 195), 34: (174, 122), 35: (86, 72), 36: (65, 85), 37: (94, 171), 38: (48, 134), 39: (106, 98), 40: (68, 121), 41: (97, 90), 42: (66, 130), 43: (115, 96), 44: (21, 188), 45: (66, 44), 46: (94, 82), 47: (86, 68), 48: (105, 95), 49: (104, 107), 
numberIntersection=0 minAngle=0.0517710787684198 sumAngle=46.10863835441612 minEdgeLength=3.0 sumEdgeLength=3081.511429247251 minDistance=2.8284271247461903 sumDistance=289672.91821577825 
Time taken: 240.706 s
